package.json

```diff
@@ -75,12 +75,12 @@
 		"prebuild-server": "check-npm-client && mkdirp build",
 		"build-server": "check-npm-client && BROWSERSLIST_ENV=server webpack --config client/webpack.config.node.js --display errors-only",
 		"build-client": "check-npm-client && yarn run build-client-evergreen",
-		"build-client-do": "check-npm-client && node ${NODE_ARGS:---max_old_space_size=8192} ./node_modules/webpack/bin/webpack.js --config client/webpack.config.js --display errors-only && yarn run build-languages-if-enabled",
+		"build-client-do": "check-npm-client && NODE_OPTIONS=${NODE_OPTIONS:---max-old-space-size=8192} node ./node_modules/webpack/bin/webpack.js --config client/webpack.config.js --display errors-only",
```

Comment:
The usage of default bash environment variable is incorrect here: `https://github.com/Automattic/wp-calypso/pull/44696/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R78`:

`node ${NODE_ARGS:---max_old_space_size=8192} ` should be `NODE_OPTIONS=${NODE_OPTIONS:---max-old-space-size=8192}`. Node expects either the raw argument `node --max-old-space-size=8192 file.js` OR the enviornment variable NODE_OPTIONS. Settings NODE_OPTIONS with `${NODE_OPTIONS:---max-old-space-size=8192}` (i.e. without left-hand assignment) is incorrect, and is easy to test. A node script run against this invocation will resolve `NODE_OPTIONS` as `undefined`.

Reply:
I'd drop any mention to `NODE_OPTIONS` here:

* If `NODE_OPTIONS` was undefined it will default to `NODE_OPTIONS=--max-old-space-size=8192 node ./node_modules/webpack/bin/webpack.js` which is not 100% correct because we can't assume that the machine running this command has 8gb of memory. A better solution is to rely on Calypso Doctor that will instruct developers to set a global value for `NODE_OPTIONS` that make sense for their machines, and we'll do the same in CircleCI config (with the extra benefit that it will help _all_ node processes, not just this one)

* If `NODE_OPTIONS` was defined this evaluates to `NODE_OPTIONS=$NODE_OPTIONS node ...` which is redundant.

Reply:
I think the intention of setting a default value here is to minimize cognitive overhead for developers trying to set up Calypso. Most devs ignore/don't know about Calypso Doctor (I know I didn't until a few days ago), and so a reasonable default is probably a good thing here.

In addition, the default value of `max-old-space-size` when unset is [2GB](https://stackoverflow.com/questions/48387040/nodejs-recommended-max-old-space-size/48392705), which will definitely kill the Calypso build.

> `NODE_OPTIONS=$NODE_OPTIONS node ...` which is redundant

🤔 We can try dropping the left-hand argument and see. (i.e. just make it `${NODE_OPTIONS:---max-old-space-size=8192}`

Reply:
> Most devs ignore/don't know about Calypso Doctor (I know I didn't until a few days ago)

Well, it was created a few days ago :D Jokes aside, I agree is not super visible. Maybe we can run it right before `yarn start` to maximize visibility?

At any rate I still think we should drop the NODE_OPTIONS here, but that's not a blocker for this PR at all.